haas/model.py

```diff
@@ -201,6 +154,7 @@ def start_console(self):
     # because we are not interested in the ouput of this command.
     @no_dry_run
     def stop_console(self):
+        return
```

Comment:
I moved the console related code to ipmi.py under obm. Did necessary changes to server.py and api.py